[External API Doc] Added rate limit endpoint section (#108)
affinity/slate: source/includes/_rate-limits.md

## API Rate Limits

> For rate limit endpoint, please refer <a href="#rate-limit">Rate Limit</a>.

Reviewer: ```suggestion
For our rate limit endpoint, please refer to <a href="#rate-limit-endpoint">Rate Limit endpoint</a>.
```

### Plan Tier Limits

> Your account plan tier will limit the number of requests each API key can make per month.

Reviewer: ```suggestion
Your account plan tier will limit the overall number of requests you can make per month.
```

| Premium       | 100,000         |

> | Enterprise    | Unlimited       |

Reviewer: Add a line here too:

This monthly account-level limit resets at the end of each calendar month.
